File: back-end/src/main/java/com/spring/restaurant/model/Order.java
package com.spring.restaurant.model;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "orders")
public class Order extends CategoryOrder {


    @Column(name = "price")
    private int price;
    @Column(name = "img")
    private String img;
    @Column(name = "description")
    private String description;


    @ManyToOne
    @JoinColumn(name = "id_category")
    private Category category;





}

File: front-end/src/app/model/order.ts
export class Order {

  id: number ;
  name: string;
  description: string;
  data_create: Date;
  data_update: Date;
  img: string;
  price: number;

  constructor(id: number, name: string, description: string, data_create: Date, data_update: Date, img: string, price: number) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.data_create = data_create;
    this.data_update = data_update;
    this.img = img;
    this.price = price;
  }



}
